tests/test_binary_quadratic_model.py

```diff
@@ -1947,3 +1947,13 @@ def test_bson_sparse(self):
         dec_bqm = dimod.BinaryQuadraticModel.from_bson(enc_bqm)
 
         self.assertEqual(bqm, dec_bqm)
+
+class TestMarkovRandomField(unittest.TestCase):
+    def test_energy_single_interaction(self):
+        edges = [(0, 1, .5, -.1, 3, 0.7)]
+        bqm = dimod.BinaryQuadraticModel.from_markov_random_field(edges)
+
+        self.assertAlmostEqual(bqm.energy({0: 1, 1: 1}), edges[0][2])
```

Comment:
Why 'almost'? Just floating point differences?

Reply:
yes

Reply:
it might actually work, just a habit for energy calculations :slightly_smiling_face: